opennms-base-assembly/src/main/filtered/etc/datacollection/ptpbase.xml

```diff
@@ -0,0 +1,218 @@
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="ptpbaseClockTimePropertiesDSEntry" label="ptpbaseClockTimePropertiesDSEntry" resourceLabel="${index}">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="ptpbaseClockTransDefaultDSEntry" label="ptpbaseClockTransDefaultDSEntry" resourceLabel="${index}">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="ptpbaseClockPortEntry" label="ptpbaseClockPortEntry" resourceLabel="${index}">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="ptpbaseClockPortDSEntry" label="ptpbaseClockPortDSEntry" resourceLabel="${index}">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="ptpbaseClockPortRunningEntry" label="ptpbaseClockPortRunningEntry" resourceLabel="${index}">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="ptpbaseClockPortTransDSEntry" label="ptpbaseClockPortTransDSEntry" resourceLabel="${index}">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="ptpbaseClockPortAssociateEntry" label="ptpbaseClockPortAssociateEntry" resourceLabel="${index}">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <group name="ptpbaseSystemTable" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.241.1.1.1.1.1" instance="ptpbaseSystemEntry" alias="ptpDomainIndex" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.1.1.1.2" instance="ptpbaseSystemEntry" alias="ptpInstanceIndex" type="integer"/>
+      <mibObj oid=".1.3.6.1.2.1.241.1.1.1.1.3" instance="ptpbaseSystemEntry" alias="ptpDomaClocPortTota" type="gauge32"/>
+   </group>
```

Comment:
We have started to separate the resource types in `resource-types.d`, we might want to do the same for these here as well.

Reply:
Not for SNMP resourceTypes. These are all still in the files under etc/datacollection/.